### ISSUE TITLE
Org-Profile:  Let's eliminate the bottom two  "No hours are pending approval"

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#843]
Seems to be some unnecessary repetition. 

![image](https://user-images.githubusercontent.com/26234440/34074463-20d4dfa8-e275-11e7-8799-9fa934ce0bb3.png)

**Comment by dannypernik** [comment]
@MaxTattenbach changed in develop and ready to test

**Comment by MaxTattenbach** [comment]
@dannypernik  lets leave at least one line - "no hours pending approval"
![image](https://user-images.githubusercontent.com/26234440/34133410-2ccd8e80-e41a-11e7-96e4-045a7951b98c.png)

**Comment by dannypernik** [comment]
Hm, that's strange. @nickolashe here's what I have:
```
<div class="left row one-margin-top full-width collapse">
    <div class="small-11 medium-10 small-centered columns">
      <h6 class="half-margin-bottom">Hours pending</h6>
      {% for admin in org_admins %}
        {% for i in hours_pending_by_admin %}
          {% if i|keyvalue:admin.user.id != None %}
            <a href="{% url 'openCurrents:hours-detail' %}?is_admin=1&amp;user_id={{admin.user.id}}&amp;type=pending"
            class="button round tiny secondary">
              {{ admin.user.first_name }} {{ admin.user.last_name }}:
              {{i|keyvalue:admin.user.id}}
            </a>
          {% endif %}
        {% endfor %}
      {% empty %}
        <p>
          No hours are currently pending approval.
        </p>
      {% endfor %}
    </div>
  </div>
```

Previously, the `{% empty %}` tag was part of the inner `for` loop and the 'no hours' message was displayed three times. I figured the empty message needed to be moved to the outer `for` loop but now it's not displaying at all. Any idea why that's happening?

**Comment by dannypernik** [comment]
@MaxTattenbach ha turns out @tevtonez had already fixed this, and I changed it. Try now please.

**Comment by dannypernik** [comment]
@nickolashe @tevtonez it seems that my commit automatically closed this issue because I had "fix #843" in the commit message. Turns out that all of these words followed by issue number will do the same: 
close
closes
closed
fix
fixes
fixed
resolve
resolves
resolved

Good to know.
